_includes/head.html

```diff
@@ -5,6 +5,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="/assets/css/styles.css">
+    <link href="/assets/css/pygments.css" rel="stylesheet">
```

Comment:
any reason why we can't include this in the scss and have it compiled into a single css file?

Reply:
No, no reason, this was my attempt to see how did they look like, but feel free to implement in the best way!